src/test/java/com/amazon/jenkins/ec2fleet/IntegrationTest.java

```diff
@@ -370,7 +370,7 @@ public Object answer(InvocationOnMock invocationOnMock) {
         final AtomicInteger targetCapacity = new AtomicInteger(0);
 
         // force to use ec2 fleet
```

Comment:
update comment to say `spot fleet`

Reply:
Also, the test names in this class should be changed to `mockFleetApiToEc2SpotFleetWithDelay ` I think.

Also, I think a few new tests should be to this class for EC2Fleet, just like how `SpotFleet` is created and tested

Reply:
What about mockFleetApiToSpotFleetWithDelay?

Reply:
Gavin, I meant new helper methods for EC2Fleet, but lets skip that for now.